recipe/meta.yaml

```diff
@@ -11,32 +11,33 @@ source:
 
 build:
   number: 0
-  skip: true  # [py<39]
-  script: {{ PYTHON }} -m pip install . -vv
+  # maturin isn't available on s390x
+  skip: true  # [py<39 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
   host:
     - python
+    - lazrs-python 0.5.0
```

Comment:
According to [this](https://github.com/laspy/laspy/blob/2.4.1/pyproject.toml), we only need `setuptools` and `wheel`. Can you please help me understand why we have `lazrs-python` in the `host` section as well as `run` when it seems to be an extra dependency according to [this](https://github.com/laspy/laspy/blob/2.4.1/setup.py#L38)?

Reply:
Please correct my understanding that if `numpy` is a dependency, then it should be listed both in the `run` and `host` section just like `python`?

Reply:
I added LAZ support intentionally, https://github.com/laspy/laspy#installation to enable LAZ backend https://github.com/laspy/laspy#features. I'll make a comment about this in the recipe

Reply:
I removed lazrs-python 0.5.0 from host